### PR TITLE
Syndie Borg fix

### PR DESCRIPTION
**Pull request by scrubmcnoob**
No longer able to see/gossip with syndie/commando borgs over PDA messenger.
#7948
:cl:
Fix:No longer able to see/gossip with syndie/commando borgs over PDA messenger.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -68,6 +68,7 @@ var/list/robot_verbs_default = list(
 	var/lockcharge //Used when locking down a borg to preserve cell charge
 	var/speed = 0 //Cause sec borgs gotta go fast //No they dont!
 	var/scrambledcodes = 0 // Used to determine if a borg shows up on the robotics console.  Setting to one hides them.
+	var/pdahide = 0 //Used to hide the borg from the messenger list
```
Comment (Birdtalon) [inline code/modules/mob/living/silicon/robot/robot.dm]: `FALSE`

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 2]
```diff
@@ -1323,6 +1326,7 @@ var/list/robot_verbs_default = list(
 	icon_state = "nano_bloodhound"
 	lawupdate = 0
 	scrambledcodes = 1
+	pdahide = 1
```
Comment (Birdtalon) [inline code/modules/mob/living/silicon/robot/robot.dm]: `TRUE`

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 3]
```diff
@@ -1371,6 +1375,7 @@ var/list/robot_verbs_default = list(
 	icon_state = "syndie_bloodhound"
 	lawupdate = 0
 	scrambledcodes = 1
+	pdahide = 1
```
Comment (Birdtalon) [inline code/modules/mob/living/silicon/robot/robot.dm]: `TRUE`

File: code/modules/mob/living/silicon/robot/robot.dm (hunk 1 above)
Comment (scrubmcnoob) [reply code/modules/mob/living/silicon/robot/robot.dm]: That wouldn't be consistent with the file though.

Comment (Birdtalon) [reply code/modules/mob/living/silicon/robot/robot.dm]: True, but we should ideally be working towards better standards when introducing newer code.

Comment (scrubmcnoob) [reply code/modules/mob/living/silicon/robot/robot.dm]: This is a bugfix solely, not a standard rewrite. I rather not complicate any merge potential with semantic Boolean values. Though, thanks for the reviewing in general. I do appreciate the thought.

**Comment by KingPhilipIII** [comment]
Feature.

**Comment by tigercat2000** [comment]
LGTM

**Pull request merged**